namespaces/init.go

```diff
@@ -141,6 +151,85 @@ func Init(container *libcontainer.Config, uncleanRootfs, consolePath string, pip
 	return system.Execv(args[0], args[0:], os.Environ())
 }
 
+func initUserNs(container *libcontainer.Config, uncleanRootfs, consolePath string, pipe *os.File, args []string) (err error) {
```

Comment:
Yeah, that's what I started with but right now at-least 2 such blocks are necessary and probably more in future. I thought it might be easier to read it if the two paths are separate.